max_sub_array/max_sub_array.py

```diff
@@ -0,0 +1,13 @@
+def max_sub_array_sum(a, size):
+    max_so_far = -9999999 - 1
+    max_ending_here = 0
+    for i in range(0, size):
+        max_ending_here = max_ending_here + a[i]
+        if (max_so_far < max_ending_here):
+            max_so_far = max_ending_here
+        if max_ending_here < 0:
+            max_ending_here = 0
+    return max_so_far
+# Driver function to check the above function  
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp7_1_ei92/max_sub_array/max_sub_array.py
+++ b/tmp/tmp7_1_ei92/max_sub_array/max_sub_array.py
@@ -8,6 +8,8 @@
         if max_ending_here < 0:
             max_ending_here = 0
     return max_so_far
-# Driver function to check the above function  
-a = [-2, -3, 4, -1, -2, 1, 5, -3] 
+
+
+# Driver function to check the above function
+a = [-2, -3, 4, -1, -2, 1, 5, -3]
 print("Maximum contiguous sum is", max_sub_array_sum(a, len(a)))
```